frontend/dashboard/components/RepoList/ActionLinks.tsx

```diff
@@ -36,52 +36,56 @@ export const ActionLinks = ({ repo }: ActionLinksProps): React.ReactElement => {
 
   const repoFullName = repo.full_name;
   const [org, repoName] = repoFullName.split('/');
-  const isDataModelRepo = repoFullName.endsWith(DATA_MODEL_REPO_IDENTIFIER);
   const editUrl = getRepoEditUrl({ org, repo: repoName });
-  const editTextKey = t(isDataModelRepo ? 'dashboard.edit_data_models' : 'dashboard.edit_app', {
-    appName: repoName,
-  });
+
+  const giteaIconWithLink = (
+    <a href={repo.html_url}>
+      <span className={cn('fa fa-gitea', classes.giteaIcon)} />
+    </a>
+  );
+
+  const editIconWithLink = (
+    <a href={editUrl}>
```

Comment:
Maybe we could use StudioLink? 🤔

Reply:
We currently don't have a `StudioLink` component. I suggest we keep using `<a>`, since we don't need any link formatting from Designsystemet on these buttons?

Reply:
Okey, that might be possible 😄 
Out of curiosity, could you attach a photo of the link buttons so we can see how they look?

Reply:
These are the ones:
![image](https://github.com/user-attachments/assets/9220bacb-9047-45d4-857b-2375afa79bdd)

Reply:
Okey, nice 😄 
And how does it look like when you hover them or access them with keyboard? Does the styling look ok then as well?

Reply:
Yep 😄 
https://github.com/user-attachments/assets/3cb96b85-4872-447a-9018-26f49edab4d6

Reply:
Perfect, looks really god! 😄